src/plugin/microphone/index.js

```diff
@@ -400,13 +400,13 @@ export default class MicrophonePlugin {
             return result;
         } else if (
             navigator.mediaDevices &&
-            navigator.userAgent.match(/(Edge\/(\d+).(\d+)$)/)
+            navigator.userAgent.match(/Edge\/\d+.\d+$/)
```

Comment:
Not really a major issue, Sonar says 
```
Make sure the regex used here, which is vulnerable to super-linear runtime due to backtracking, cannot lead to denial of service.
```
This is probably for server side node more than client side, but I updated the Regex to avoid this error. So now it return one match instead of 3

Reply:
LGTM